.github/workflows/main.yml

```
@@ -45,6 +45,8 @@ jobs:
       - name: Install Embedded Arm Toolchain arm-none-eabi-gcc
         if:   steps.cache-toolchain.outputs.cache-hit != 'true'  # Install toolchain if not found in cache
         uses: fiam/arm-none-eabi-gcc@v1.0.2
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Upgrading the version avoid using the unsecure flag:
```suggestion
        uses: fiam/arm-none-eabi-gcc@v1.0.4
```

(See https://github.com/JF002/InfiniTime/pull/534/files#r678790589)